lab_1_classify_by_unigrams/main.py

```diff
@@ -33,11 +54,17 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
         dict[str, float] | None: A dictionary with frequencies
 
     In case of corrupt input arguments, None is returned
-    """
+"""
 
 
 def create_language_profile(language: str, text: str) -> dict[str, str | dict[str, float]] | None:
-    """
+    if isinstance(language, str) and isinstance(text, str):
+        lang_profile_dict = {'name': language, 'freq': calculate_frequencies(tokenize(text))}
```

Comment:
you can return dict "as is", without creating the variable

Reply:
done